train_and_eval/eval/eval_MRL.py

```diff
@@ -49,60 +52,32 @@
 elif args.dataset == 'A':
```

Comment:
We should perhaps add links to these datasets or relevant papers in the header, or describe them with little more verbosity